src/run.rs

```diff
@@ -395,7 +395,14 @@ pub fn run(settings: &Settings, state: &mut State) -> io::Result<()> {
                     continue;
                 }
             } else if event.r#type == "image"
-                && (event.action == "import" || event.action == "load" || event.action == "pull")
+                && (event.action == "delete"
+                    || event.action == "import"
+                    || event.action == "load"
+                    || event.action == "pull"
+                    || event.action == "push"
+                    || event.action == "save"
+                    || event.action == "tag"
+                    || event.action == "untag")
```

Comment:
Looks like we can ignore `"delete"` or `"untag"`?

Reply:
That's a good point. I'll cut a new version.